### PR TITLE
Update data.json

### PR DESCRIPTION
**Pull request by mohitrakhade20**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -341,5 +341,10 @@
     "githubUsername": "rosajen27",
     "imageUrl": "https://raw.githubusercontent.com/rosajen27/rosajen27/master/rosajen27.png",
     "issueNumber": 165
+       },
+	{
+    "name": "Mohit Rakhade",
+    "githubUsername": "mohitrakhade20"
+
```
Comment (eddiejaoude) [inline data.json:348]: You need an `imageUrl` and `issueNumber`

**Comment by mohitrakhade20** [comment]
how to get img URL?

**Comment by eddiejaoude** [comment]
From the documentation...

> To add your github profile, create an issue, with the title as your name, and in the description add a screenshot of the customised GitHub profile page, copy the generated image url

**Comment by mohitrakhade20** [comment]
You can merge this PR now

File: data.json [hunk 2]
```diff
@@ -347,10 +347,19 @@
     "githubUsername": "rosajen27",
     "imageUrl": "https://raw.githubusercontent.com/rosajen27/rosajen27/master/rosajen27.png",
     "issueNumber": 165
-  },	{
+  },
+	{
```
Comment (eddiejaoude) [inline data.json:351]: Formatting is not consistent, but this can be fixed in another PR

File: data.json [hunk 3]
```diff
@@ -347,10 +347,19 @@
...
+    "githubUsername": "mohitrakhade20",
+    "imageUrl": "https://user-images.githubusercontent.com/48031017/98556319-eecabe00-22c8-11eb-8692-57d2efa3d3ab.png",
+    "issueNumber": 168
+  },
+  {
     "name": "Jayant Goel",
     "githubUsername": "JayantGoel001",
     "imageUrl": "https://user-images.githubusercontent.com/54479676/98487992-88a05580-224c-11eb-8e76-d8a59aa05bd4.png",
     "issueNumber": 169
+
```
Comment (eddiejaoude) [inline data.json:362]: Not sure what the extra blank line is needed?

Comment (mohitrakhade20) [reply data.json]: please check now

**Comment by mohitrakhade20** [comment]
please check is it okay now?

File: data.json [hunk 4]
```diff
@@ -316,43 +316,46 @@
...
+    "githubUsername": "mohitrakhade20",
+    "imageUrl": "https://user-images.githubusercontent.com/48031017/98556319-eecabe00-22c8-11eb-8692-57d2efa3d3ab.png",
+    "issueNumber": 168
+  },
+  {
+    "name": "Jayant Goel",
+    "githubUsername": "JayantGoel001",
+    "imageUrl": "https://user-images.githubusercontent.com/54479676/98487992-88a05580-224c-11eb-8e76-d8a59aa05bd4.png",
+    "issueNumber": 169
+ }
```
Comment (eddiejaoude) [inline data.json:361]: Looking better, but you removed 1 too many blank lines - the closing blank line at the end of the file is still needed

Comment (mohitrakhade20) [reply data.json]: I added it

Comment (eddiejaoude) [inline data.json:361]: I just noticed that the closing array / collection bracket is now missing. The objects need to be closed with a `]`

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**